readme.md

```diff
@@ -15,6 +15,17 @@ Ultimate success would be to merge this functionality into periph.io lib.
 - may leak `req.fd` descriptors, TODO test
 - 64-bit system likely have different ioctl numbers, please try and write me back
 
+# Testing:
```

Comment:
Please make it just `# Testing` because punctuation (`:` `.` etc) is not used in headers.